services/core-web/src/components/noticeOfWork/applications/review/ReviewNOWApplication.js

```diff
@@ -473,6 +473,7 @@ export const ReviewNOWApplication = (props) => {
             Access presently gated
             {props.isPreLaunch && <NOWFieldOriginTooltip />}
             <NOWOriginalValueTooltip
+              style={{ marginLeft: "5%" }}
               originalValue={props.renderOriginalValues("is_access_gated").value}
```

Comment:
There is a reason I didn't use a class for this - 
NOWOriginalValueTooltip expects a style prop, and then passes that prop through as `style={{props.style.marginLeft}}`, This needs to be a style prop to override the default class on the component. 

I could put the '5%' into a constant, but it would still be passed through in the same way 
`style={{ marginLeft: CONSTANT_FIVE_PERCENT }}` doesn't seem to clean it up much.